modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/io/Clients.scala

```diff
@@ -56,8 +58,17 @@ object Clients {
       }
     }
 
-  def mkHttp[F[_]: ConcurrentEffect](ec: ExecutionContext): Resource[F, HttpClient[F]] =
-    BlazeClientBuilder[F](ec).resource
+  def mkHttp[F[_]: ConcurrentEffect](
+    connectionTimeout: FiniteDuration = defaults.ConnectTimeout,
+    readTimeout: FiniteDuration = defaults.RequestTimeout,
+    maxConnections: Int = 10, // http4s uses 10 by default
```

Comment:
It would be awesome to make `connectionTimeout`, `readTimeout` and `maxConnections` configurable with defaults and to explain in the config file what the HTTP client is used for.

I know that we discussed using the lib defaults vs using our own. Here already we are using one of our own (`10`) that is impossible to update. What if we want to update the timeouts also? I'm strongly in favor of putting these in `application.conf`. Default timeouts from the lib will probably never change and even if they do and we didn't update them in `application.conf`, well, they are just defaults and can still be updated in the `config.hocon`.